assets/js/state/sagas/index.js

```diff
@@ -124,6 +126,8 @@ function* initialDataFetch() {
   const { data: databases } = yield call(get, '/databases');
   yield put(setDatabases(databases));
   yield put(stopDatabasesLoading());
+
+  yield put(fetchSoftwareUpdatesSettings());
```

Comment:
Do we need this when doing initial fetching? Maybe we could just dispatch the refresh when loading the about page, what do you think?

Reply:
I don't actually mind. I just thought that we might not need to always call the endpoint when landing on the settings.

I can remove that and let the settings page dispatch a `fetchSoftwareUpdatesSettings`, ok?

Reply:
Yes! :+1: